OpenRA.Mods.Common/Traits/World/EditorCursorLayer.cs

```diff
@@ -107,7 +107,7 @@ void ITickRender.TickRender(WorldRenderer wr, Actor self)
 					if (Actor.RemoveInits<SubCellInit>() > 0)
 						updated = true;
 
-					var subcell = world.Map.Tiles.Contains(cell) ? editorLayer.FreeSubCellAt(cell) : SubCell.Invalid;
+					var subcell = world.Map.Contains(cell) ? editorLayer.FreeSubCellAt(cell) : SubCell.Invalid;
```

Comment:
```suggestion
					var subcell = ((IMapTiles)world.Map).Tiles.Contains(cell) ? editorLayer.FreeSubCellAt(cell) : SubCell.Invalid;
```

?

Reply:
Done, wasn't sure they were equivalent but yes this works.